### PR TITLE
Add initial set of files to implement running Flask skeleton

### PR DESCRIPTION
**Pull request by jonmorphic**
Initial creation of basic service, which just implements the info endpoint.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: gunicorn app:app
```
Comment (pricem14pc) [inline Procfile]: Hi @jonmorphic is this a new python webserver or are we using it for any of our other apps?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from setuptools import setup, find_packages
+
+NAME = "ras_backstage"
+VERSION = "0.0.2"
```
Comment (pricem14pc) [inline setup.py]: Does this need to align with the config.yaml VERSION?

File: Procfile (hunk 1 above)
Comment (jonmorphic) [inline Procfile]: It's the same wsgi server as rad-party

File: setup.py (hunk 2 above)
Comment (jonmorphic) [inline setup.py]: Yes, good spot :)

**Comment by jonmorphic** [comment]
Agree, will configure the Jenkins jobs first thing and get it deployed

**Pull request merged**